alibi_detect/utils/_types.py

```diff
@@ -7,9 +7,9 @@
 
 # Literal for typing
 if sys.version_info >= (3, 8):
-    from typing import Literal  # noqa
+    from typing import Literal, Protocol, runtime_checkable  # noqa
 else:
-    from typing_extensions import Literal  # noqa
+    from typing_extensions import Literal, Protocol, runtime_checkable # noqa
```

Comment:
Can remove since they're imported directly from `typing_extensions` in `od.base`?